src/Microsoft.Azure.WebJobs.Host/InvokeFunctionFilterAttribute.cs

```
@@ -0,0 +1,128 @@
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.WebJobs.Host
+{
+    /// <summary>
+    /// An invocation filter that invokes job methods
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Class | AttributeTargets.Method, Inherited = true, AllowMultiple = true)]
+    public sealed class InvokeFunctionFilterAttribute : InvocationFilterAttribute, IFunctionInvocationFilter
+    {
+        /// <summary>
+        /// When this attribute is used, take the name of the function to invoke
+        /// </summary>
+        /// <param name="executingFilter"></param>
+        /// <param name="executedFilter"></param>
+        public InvokeFunctionFilterAttribute(string executingFilter = null, string executedFilter = null)
+        {
+            ExecutingFilter = executingFilter;
+            ExecutedFilter = executedFilter;
+        }
+
+        /// <summary>
+        /// Executing filter
+        /// </summary>
+        public string ExecutingFilter { get; }
+
+        /// <summary>
+        /// Executed filter
+        /// </summary>
+        public string ExecutedFilter { get; }
+
+        /// <summary>
+        /// Call the requested job method before the main function call
+        /// </summary>
+        /// <returns></returns>
+        public override async Task OnExecutingAsync(FunctionExecutingContext executingContext, CancellationToken cancellationToken)
+        {
+            if (executingContext == null)
+            {
+                throw new ArgumentNullException("executingContext");
+            }
+            
+            if (!string.IsNullOrEmpty(ExecutingFilter))
+            {
+                executingContext.Logger.LogInformation("Executing Function Filter '" + ExecutingFilter + "'");
+
+                try
```

Remove this try/catch block (don't swallow exceptions)